src/devices/Mcp3xxx/Mcp33xx.cs

```diff
@@ -0,0 +1,75 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.Spi;
+
+namespace Iot.Device.Adc
+{
+    /// <summary>
+    /// MCP3304 Analog to Digital Converter (ADC)
+    /// </summary>
+    public abstract class Mcp33xx : Mcp3xxx
+    {
+        /// <summary>
+        /// Constructs Mcp33xx instance
+        /// </summary>
+        /// <param name="spiDevice">Device used for SPI communication</param>
+        /// <param name="channelCount">Value representing the number of single ended input channels on the device.</param>
+        /// <param name="adcResolutionBits">The number of bits of resolution for the ADC.</param>
+        public Mcp33xx(SpiDevice spiDevice, byte channelCount, byte adcResolutionBits) : base(spiDevice, channelCount, adcResolutionBits) { }
+
+        /// <summary>
+        /// Reads a  value from the device using pseudo-differential inputs
+        /// </summary>
+        /// <param name="valueChannel">Channel which represents the signal (valid values: 0 to channelcount - 1).</param>
+        /// <param name="referenceChannel">Channel which represents the signal ground (valid values: 0 to channelcount - 1).</param>
+        /// <returns>A value corresponding to relative voltage level on specified device channels</returns>
+        public override int ReadPseudoDifferential(int valueChannel, int referenceChannel)
+        {
+            throw new NotSupportedException("Mcp33xx device does not support ReadPseudoDifferential.");
```

Comment:
`$"..... {nameof(ReadPseudoDifferential)} ....."` - this is making it easier to rename this if we decide to do so in the future (editor will do it for you)